app/messaging/publish-debt-payment-request.js

```diff
@@ -1,34 +1,49 @@
-const util = require('util')
-const createMessage = require('./create-message')
-const { updateQualityChecksStatus } = require('../quality-check')
-const {
-  getDebtPaymentRequests,
-  updatePaymentRequestReleased
-} = require('../payment-request')
-
-const publishDebtPaymentRequests = async (debtSender) => {
-  try {
-    const debtPaymentRequests = await getDebtPaymentRequests()
-    for (const paymentRequest of debtPaymentRequests) {
-      const { paymentRequestId } = paymentRequest
-      delete paymentRequest.paymentRequestId
-      await publishPaymentRequest(paymentRequest, debtSender)
-      await updatePaymentRequestReleased(paymentRequestId)
-      await updateQualityChecksStatus(paymentRequestId, 'Processed')
-    }
-  } catch (err) {
-    console.error('Unable to process payment request message:', err)
-  }
-}
-
-const publishPaymentRequest = async (paymentRequest, debtSender) => {
-  const message = createMessage(paymentRequest, 'uk.gov.pay.debt.check')
-  await debtSender.sendMessage(message)
-
-  console.log('Completed request sent:', util.inspect(message, false, null, true))
-}
-
-module.exports = {
-  publishDebtPaymentRequests,
-  publishPaymentRequest
-}
+const db = require('../data')
+const util = require('util')
+const createMessage = require('./create-message')
+const { updateQualityChecksStatus } = require('../quality-check')
+const { AWAITING_ENRICHMENT, PASSED } = require('../quality-check/statuses')
+const {
+  getDebtPaymentRequests,
+  updatePaymentRequestReleased,
+  updatePaymentRequestCategory
+} = require('../payment-request')
+
+const publishDebtPaymentRequests = async (debtSender) => {
+  try {
+    const debtPaymentRequests = await getDebtPaymentRequests()
+    for (const paymentRequest of debtPaymentRequests) {
+      const { paymentRequestId } = paymentRequest
+      const inManualLedgerAwaitingDebtData = await db.qualityCheck.findOne({
+        where: {
+          paymentRequestId,
+          status: AWAITING_ENRICHMENT
+        }
+      })
+
+      if (inManualLedgerAwaitingDebtData) {
+        await updatePaymentRequestCategory(paymentRequestId, 2)

```

Comment:
Create a constant module for payment request categories like done for statuses (I have no idea what category 2 is for example)